index.html

```diff
@@ -64,18 +65,18 @@
     <!-- Header -->
     <header id="top" class="header intro">
         <div class="text-vertical-center">
-            <h1>SIMS</h1>
-            <h3>Surge Information Management Support</h3>
+            <h1 class="front-page-font-outline">SIMS</h1>
+            <h3 class="front-page-font-outline">Surge Information Management Support</h3>
             <br>
             <a href="#about" class="btn btn-dark btn-lg">Find Out More</a>
         </div>
     </header>
 
     <!-- About -->
-    <section id="about" class="about">
+    <section id="about" class="about bg-white">
```

Comment:
I don't like the zebra approach to the site now, white/black is to big of a contrast. If you want to try this try white and a soft grey or other soft color.